templates/nginx.conf

```diff
@@ -70,6 +71,22 @@ server {
 
       rewrite ^(.+)/+$ $1 permanent;
 
+      #match /static and /css paths
+      location ~ ^/(static|css) {
+        proxy_cache lemmy_static_cache_{{domain}};
+        proxy_cache_use_stale error updating timeout http_500 http_502 http_503 http_504;
+
+        proxy_cache_revalidate on;
+        proxy_cache_background_update on;
+        proxy_cache_lock on;
+
+        proxy_cache_valid 200 1d; #only cache 200s
+
+        proxy_ignore_headers Cache-Control;
+        add_header Cache-Control "public, max-age=86400, stale-while-revalidate=10" always;  #cache for 1 day
+        add_header X-Cache-Status $upstream_cache_status;
+      }
```

Comment:
Its better to fix these things in the respective repos so that they send proper cache-control headers. Putting it here makes things unnecessarily complicated.